### DISCUSSION TITLE
`slither .` command is checking all contracts in lib folder along with the PuppyRaffle.sol (Section 4)

### DISCUSSION THREAD

**Discussion by KatrixReloaded** [Cyfrin/security-and-auditing-full-course-s23#163] [Q&A] [answered]
It is checking for vulnerabilities in openzeppelin-contracts and forge-std too, which leads to it generating a huge output which is not even entirely visible in my terminal.  

Output:
```
INFO:Slither:. analyzed (51 contracts with 85 detectors), 675 result(s) found
```

**Comment by EngrPips** [comment, marked as answer]
Hello @KatrixReloaded, you should set up a `slither.config.json` in your directory and put the below configuration in it. It shouldn't check vulnerabilities in dependencies anymore. You can also tweak the configurations to your requirements.

```javascript
{
  "detectors_to_exclude": "conformance-to-solidity-naming-conventions,solc-version",
  "exclude_informational": false,
  "exclude_low": false,
  "exclude_medium": false,
  "exclude_high": false,
  "disable_color": false,
  "filter_paths": "(mocks/|test/|script/|upgradedProtocol/)",
  "legacy_ast": false,
  "exclude_dependencies": true
}
```

**Reply by KatrixReloaded** [reply]
Thanks! It worked.

**Reply by EngrPips** [reply]
Awesome, I'm glad you fixed it. Keep crushing it champ.
